ord_schema/interface/query_test.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 Open Reaction Database Project Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for ord_schema.interface.query."""
```

Comment:
Do we want to exclude `interface` from `run_tests.sh`? I don't want the list of dependencies to get too bloated since not everyone will care about spinning up their own postgres server or installing docker.

Reply:
The docker python bindings are pretty lightweight, but this test does add significantly to the runtime for run_tests.sh. I'll leave it out for now and think about a better test manager script for a future PR.

Reply:
PTAL

Reply:
LGTM; thanks for separating